include/deal.II/lac/block_vector_base.h

```diff
@@ -2264,6 +2262,17 @@ void BlockVectorBase<VectorType>::extract_subvector_to (ForwardIterator
     }
 }
 
+
+template <typename VectorType>
+inline
+bool BlockVectorBase<VectorType>::supports_distributed_data() const
+{
+  // Create a dummy vector of type VectorType to check
+  // supports_distributed_data even if BlockVectorBase contains no block
+  VectorType dummy;
+  return dummy.supports_distributed_data();
```

Comment:
@Rombur I agree with @bangerth here. If ```VectorType``` is an entity with a dynamic ```supports_distributed_data``` property, you have to check every block on runtime.

If this property, on the other hand is type bound, I would suggest, to either use a static member function, or - better - a type trait:
```
SupportsDistributedData<VectorType>::value
```